### PR TITLE
docs: add php supply chain security incident

### LINKED ISSUE #601: [Suggestion] PHP repository compromise missing?
Looks like we're missing out on the PHP supply chain incident?
Source: https://www.bleepingcomputer.com/news/security/phps-git-server-hacked-to-add-backdoors-to-php-source-code/

### PR DESCRIPTION
**Pull request by lirantal**
Adding the PHP supply chain security incident to the list of 2021 

Fixes #601

### CODE REVIEW

File: supply-chain-security/compromises/2021/php.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PHP is a web development language primarily used for creating web applications. On March 28th, 2021, they reported [1] that the self-hosted Git server at git.php.net was compromised and included two commits with malicious intent to add backdoor capabilities.
+
+## Impact
+
+* Code changes in the form of two commits to the official and self-hosted Git server were made as if they were signed-off by PHP maintainers Rasmus Lerdorf and Nikita Popov.
+* Code changes were introducing a remote code execution backdoor to any PHP server built from the source-code that included these commits.
+
+## Remediation
+
+* Write access to the repository's source migrated out of a self-built karma system and now restricted only to members of the PHP organization.
```
Comment (joshuagl) [inline supply-chain-security/compromises/2021/php.md:12]: Can we expand on organisation? Is that the GitHub Organisation? Or some other mechanism for identifying developers? If it is GitHub, we should probably move the switch to using GitHub to be the first item in the list.

Comment (lirantal) [reply supply-chain-security/compromises/2021/php.md:12]: Indeed, will update.

Comment (lirantal) [reply supply-chain-security/compromises/2021/php.md]: Resolved.

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
@lirantal please update the `compromises/README.md` with this incident so that its on the main page THANK YOU FOR DOING THIS!

**Comment by lirantal** [comment]
@TheFoxAtWork apologies on the delay, only seeing your reply now :-)
Will accommodate the updates.

**Comment by lirantal** [comment]
@TheFoxAtWork up to date now with the README change as well

**Review by TheFoxAtWork — APPROVED** [review]
LGTM thank you for tackling this!!

**Comment by lirantal** [comment]
Thanks Emily!

**Pull request merged**